ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -158,6 +158,7 @@ data:
               - container
               - host
               - pod
+              - vm
```

Comment:
You may want to also add to the list in the section below, it looks to me like pods in openshift- namespaces won't get the vm label right now.

Reply:
as far as I know, we probably won't have KubeVirt VM pods in `openshift` namespaces, but let me double-check with @sradco and @tiraboschi if this is needed

Reply:
added @dgoodwin